linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive_0/rule.yml

```diff
@@ -0,0 +1,71 @@
+documentation_complete: true
+
+title: 'Set SSH Client Alive Count Max'
+
+description: |-
+    The SSH server sends at most <tt>ClientAliveCountMax</tt> messages
+    during a SSH session and waits for a response from the SSH client.
+    The option <tt>ClientAliveInterval</tt> configures timeout after
+    each <tt>ClientAliveCountMax</tt> message. If the SSH server does not
+    receive a response from the client, then the connection is considered idle
+    and terminated.
+
+    To ensure the SSH idle timeout occurs precisely when the
+    <tt>ClientAliveInterval</tt> is set, set the <tt>ClientAliveCountMax</tt> to
+    value of <tt>0</tt>.
+
+rationale: |-
+    This ensures a user login will be terminated as soon as the <tt>ClientAliveInterval</tt>
+    is reached.
+
+severity: medium
+
+identifiers:
+    cce@rhel7: CCE-27082-7
+    cce@rhel8: CCE-80907-9
+    cce@rhcos4: CCE-82464-9
+    cce@sle12: CCE-83034-9
```

Comment:
Those CCEs are duplicated in the parametrized rule. Please delete four random CCEs from https://github.com/ComplianceAsCode/content/blob/master/shared/references/cce-redhat-avail.txt and use them as replacement for these ones.
This will make the CI to pass.